_includes/header.html

```diff
@@ -6,6 +6,9 @@
                     <div class="col-md-8 col-md-offset-2">
                         <h1 class="brand-heading">Konstructs</h1>
                         <p class="intro-text">An open source Infiniminer/Minecraft inspired multiplayer game.</p>
+                        <p>
+                          <iframe width="280" height="158" src="https://www.youtube.com/embed/8HmZ1Rk0Kak" frameborder="0" allowfullscreen></iframe>
```

Comment:
Seems very tricky to make a iframe resize after the content is loaded. What you recommended didn't work. Without the video I can not make the page smaller than 320 and with the video it is the same. Maybe it doesn't impact the minimal width and it is rather dictated elsewhere?